charts/heureka/docker/maintenance/scripts/maintenance.sql

```diff
@@ -0,0 +1,49 @@
+-- SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Maintenance script for database truncation
+-- Enable logging
+SET @start_time = NOW();
+SELECT CONCAT('Starting maintenance at ', @start_time) AS log_message;
+
+-- Start transaction
+START TRANSACTION;
+
+-- Disable foreign key checks temporarily
+SET FOREIGN_KEY_CHECKS = 0;
+
+-- Truncate junction join tables first
+TRUNCATE TABLE IssueMatchEvidence;
+TRUNCATE TABLE ComponentVersionIssue;
+TRUNCATE TABLE IssueRepositoryService;
+TRUNCATE TABLE ActivityHasIssue;
+TRUNCATE TABLE ActivityHasService;
+TRUNCATE TABLE IssueMatchChange;
+TRUNCATE TABLE SupportGroupService;
+TRUNCATE TABLE SupportGroupUser;
+TRUNCATE TABLE Owner;
+
+-- Truncate dependent entity tables
+TRUNCATE TABLE IssueMatch;
+TRUNCATE TABLE Evidence;
+TRUNCATE TABLE ComponentInstance;
+TRUNCATE TABLE IssueVariant;
+TRUNCATE TABLE ComponentVersion;
```

Comment:
```suggestion
TRUNCATE TABLE IssueMatch;
TRUNCATE TABLE Evidence;
TRUNCATE TABLE ComponentInstance;
TRUNCATE TABLE IssueVariant;
TRUNCATE TABLE ComponentVersion;
```

Not sure about these ones.